benchmarks/x_benchmark_blasfeo_api.c

```diff
@@ -141,8 +143,12 @@ int main()
 	const double flops_max = 8; // 1x128 bit fma
 #elif defined(TARGET_ARMV7A_ARM_CORTEX_A7)
 	const double flops_max = 2; // 1x32 bit fma
+#elif defined(TARGET_ARMV7A_ARM_CORTEX_A7)
```

Comment:
replicated Cortex A7, probably you wanted A9 instead

Reply:
Yes, indeed, and `	const double flops_max = 4; // 1x64 bit fma`, right (I don't understand where this is coming from exactly TBH)